server/sonar-server/src/main/java/org/sonar/server/platform/platformlevel/PlatformLevel4.java

```diff
@@ -230,7 +231,8 @@ protected void configureLevel() {
 
     add(
       PluginDownloader.class,
-      Views.class,
+      DeprecatedViews.class,
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this use of "DeprecatedViews"; it is deprecated. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)